config.toml

```
@@ -43,10 +43,14 @@ password = "scratch"
 max_open_conns = 3
 max_idle_conns = 1
 conn_max_lifetime = 3600
-hosted_databases = ["scratchdb"]
+hosted_databases = ["scratch"]
+hosted_clusters = ["scratch"]
 
 [[users]]
 api_key = "local"
-db_name = "scratchdb"
+db_name = "scratch"
 db_user = "scratch"
 db_password = "scratch"
+db_cluster = "scratch"
+db_shard = "scratch_shard"
+db_replica = "scratch_replica"
```

Should be: 

```suggestion
db_shard_macro = "scratch_shard"
db_replica_macro = "scratch_replica"
```

We're not specifying the shard/replica name, we're specifying the name of the macro which holds the shard/replica.